frontend/src/ui/Blocks/Playlist.tsx

```diff
@@ -0,0 +1,132 @@
+        showTitle
+        showMetadata
+        order
+        layout
+    }
+`;
+
+const playlistFragment = graphql`
+    fragment PlaylistBlockPlaylistData on Playlist {
+        __typename
+        ... on NotAllowed { dummy } # workaround
+        ... on AuthorizedPlaylist {
+            id
+            title
+            description
+            entries {
+                __typename
+                ... on AuthorizedEvent { id, ...VideoListEventData }
+                ... on Missing { dummy }
+                ... on NotAllowed { dummy }
+            }
+        }
+    }
+`;
+
+type FromBlockProps = SharedProps & {
+    fragRef: PlaylistBlockData$key;
+}
+
+export const PlaylistBlockFromBlock: React.FC<FromBlockProps> = ({ fragRef, ...rest }) => {
+    const { t } = useTranslation();
+    const { playlist, ...block } = useFragment(blockFragment, fragRef);
+    return playlist == null
+        ? <Card kind="error">{t("playlist.deleted-playlist-block")}</Card>
+        : <PlaylistBlockFromPlaylist fragRef={playlist} {...rest} {...block} />;
+};
+
+type BlockProps = Partial<Omit<Fields<PlaylistBlockData$data>, "playlist">>;
+
+type SharedFromPlaylistProps = SharedProps & BlockProps & {
+    title?: string;
+};
+
+type FromPlaylistProps = SharedFromPlaylistProps & {
+    fragRef: PlaylistBlockPlaylistData$key;
+};
+
+export const PlaylistBlockFromPlaylist: React.FC<FromPlaylistProps> = (
+    { fragRef, ...rest },
+) => {
+    const playlist = useFragment(playlistFragment, fragRef);
+    return <PlaylistBlock playlist={playlist} {...rest} />;
+};
+
+type Props = SharedFromPlaylistProps & {
+    playlist: PlaylistBlockPlaylistData$data;
+};
+
+export const PlaylistBlock: React.FC<Props> = ({ playlist, ...props }) => {
+    const { t } = useTranslation();
+
+    if (!playlist) {
+        return <VideoListBlockContainer showViewOptions={false}>
+            {t("playlist.deleted-playlist-block")}
+        </VideoListBlockContainer>;
+    }
+
+    if (playlist.__typename === "NotAllowed") {
+        return <VideoListBlockContainer showViewOptions={false}>
+            {t("playlist.not-allowed-playlist-block")}
+        </VideoListBlockContainer>;
+    }
+    if (playlist.__typename !== "AuthorizedPlaylist") {
+        return unreachable();
+    }
+
+    const items = playlist.entries.map(entry => {
+        if (entry.__typename === "AuthorizedEvent") {
+            const out = readInlineData<VideoListEventData$key>(videoListEventFragment, entry);
+            return out;
+        } else if (entry.__typename === "Missing") {
+            return "missing";
+        } else if (entry.__typename === "NotAllowed") {
+            return "unauthorized";
+        } else {
+            return unreachable();
+        }
+    });
+
+    let title = playlist.title ?? (props.showTitle ? playlist.title : undefined);
+    if (props.moreOfTitle) {
+        title = t("video.more-from-playlist", { playlist: playlist.title });
+    }
```

Comment:
That's still fishy. On test deployment u cannot not show the title due to this (probably) copy paste mistake.

```suggestion
    const title = props.showTitle
        ? (props.moreOfTitle ? t("video.more-from-playlist", { playlist: playlist.title }) : playlist.title)
        : undefined;
```